lib/ecto/migrator.ex

```diff
@@ -392,7 +392,7 @@ defmodule Ecto.Migrator do
 
   # This function will match directories passed into `Migrator.run`.
   defp migrations_for(migration_source) when is_binary(migration_source) do
-    query = Path.join(migration_source, "*")
+    query = Path.join([migration_source, "**", "*.exs"])
```

Comment:
we should only care about ".exs" files anyway, so added the pattern here, and removed [here](https://github.com/elixir-ecto/ecto_sql/pull/67/files#diff-c160bf108a9ad548fa1f2a8280bad4e2L412)